sheetwork/core/sheetwork.py

```diff
@@ -117,6 +125,20 @@ def load_sheet(self):
         logger.debug(f"Loaded SHEET HEAD: {df}")
         self.sheet_df = df
 
+    def count_empty_headers(self, df: pandas.DataFrame) -> int:
```

Comment:
not sure if the `-> int` is needed

Reply:
I think we might want to call this function
`count_empty_header_cols_in_df()`

And I think it would make sense to have it as an independent function that lives in `utils.py` that way it can be used by other parts of the tool and is also easier to test independently